examples/dev_coach_review_prd.py

```
@@ -0,0 +1,846 @@
+        if section.get("type") == "text":
+            content = section.get("content", "").strip()
+            if content:
+                text_parts.append(content)
+    return "\n".join(text_parts) if text_parts else response_data.get("raw", "")[:500]
+
+
+def log_to_file(log_file: str, label: str, content: str) -> None:
+    """Append content to log file."""
+    with open(log_file, "a") as f:
+        f.write(f"\n{'=' * 80}\n")
+        f.write(f"[{datetime.now().isoformat()}] {label}\n")
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)